packages/esm-service-queues-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -102,21 +100,35 @@ interface PaginationData {
 }
 
 function ActiveVisitsTable() {
-  const currentServiceName = useSelectedServiceName();
+  const selectedQueueUuid = useSelectedServiceUuid();
   const currentLocationUuid = useSelectedQueueLocationUuid();
-  const { visitQueueEntries, isLoading } = useVisitQueueEntries(currentServiceName, currentLocationUuid);
+  const { queueEntries, isLoading, error } = useQueueEntries({
+    queue: selectedQueueUuid,
+    location: currentLocationUuid,
+    isEnded: false,
+  });
   const useNewActiveVisitsTable = useFeatureFlag('new-queue-table');
   const layout = useLayoutType();
   const { t } = useTranslation();
-  const selectedServiceUuid = useSelectedServiceUuid();
 
-  const queueEntries = visitQueueEntries?.map((entry) => entry.queueEntry);
+  const [showOverlay, setShowOverlay] = useState(false);
+  const [viewState, setViewState] = useState<{ selectedPatientUuid: string }>(null);
+
+  useEffect(() => {
+    if (error?.message) {
+      showSnackbar({
+        title: t('errorLoadingQueueEntries', 'Error loading queue entries'),
+        kind: 'error',
+        subtitle: error?.message,
+      });
+    }
+  }, [error?.message]);
+
   if (isLoading) {
     return <DataTableSkeleton role="progressbar" />;
   } else if (useNewActiveVisitsTable) {
-    const queueEntriesInSelectedQueue = queueEntries.filter(
-      (queueEntry) => !selectedServiceUuid || queueEntry.queue.uuid == selectedServiceUuid,
-    );
+    const queueEntriesInSelectedQueue =
+      queueEntries?.filter((queueEntry) => !selectedQueueUuid || queueEntry.queue.uuid == selectedQueueUuid) ?? [];
```

Comment:
Is this still needed?  Aren't the queueEntries already filtered by selectedQueueUuid at this point, based on the parameters sent to `useQueueEntries`?  Can we get rid of this filtering?

Reply:
Yeah, I think so. Good catch.